myproject/myapp/admin.py

```diff
@@ -0,0 +1,29 @@
+
+
+from django.contrib import admin
+from .models import Client, Product, Order
+
+class ClientAdmin(admin.ModelAdmin):

```

Comment:
Можно регистрировать иначе

```suggestion
from django.contrib import admin
from .models import Client, Product, Order


@admin.register(Client)
class ClientAdmin(admin.ModelAdmin):
```